transcendence-app/tsconfig.json

```diff
@@ -15,6 +15,6 @@
     "forceConsistentCasingInFileNames": false,
     "noFallthroughCasesInSwitch": false,
     "strict": true,
-    "typeRoots": ["./node_modules/@types", "./types"]
+    "typeRoots": ["./node_modules/@types", "../node_modules/@types", "./types"]
```

Comment:
do we still need this one `"./node_modules/@types"` ?

Reply:
done 👍 

Just a warning, you may still find `node_modules` inside each workspace. Not 100% sure why this happens but it seems related with version mismatch of a same package between workspaces and root node modules. Despite having the same module with the same version defined in `package.json` there might be some transitive dependencies which don't match the package version -> https://stackoverflow.com/questions/71048697/npm-workspaces-some-modules-installed-to-local-node-modules-dir